.github/workflows/testing.yaml

```diff
@@ -0,0 +1,22 @@
+name: Testing
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Testing Job
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Setup Go
+      id: setup-go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
```

Comment:
This might be problematic with certs (as you saw with the SAN) - we know some of our users don't use SANs in their certs.